### PR TITLE
Pushing the operator image to a container registry

### PR DESCRIPTION
**Pull request by baijum**
Resolves: https://jira.coreos.com/browse/ODC-358

### CODE REVIEW

File: make/Makefile.dev [hunk 1]
```diff
@@ -38,6 +38,12 @@ create-cr:
 build-operator-image:
 	operator-sdk build $(DEVOPSCONSOLE_OPERATOR_IMAGE)
 
+.PHONY: push-operator-image
+## Push the operator container image to a container registry
+push-operator-image: build-operator-image
+	docker login -u $(QUAY_USERNAME) -p $(QUAY_PASSWORD) "quay.io"
```
Comment (corinnekrych) [inline make/Makefile.dev]: Maybe reuse `REGISTRY_URI = quay.io` from  https://github.com/redhat-developer/devopsconsole-operator/blob/master/Makefile#L60 
We've got a bit of duplication between those 2 makefiles

File: make/Makefile.dev [hunk 2]
```diff
@@ -38,6 +38,12 @@ create-cr:
...
+	docker push $(DEVOPSCONSOLE_OPERATOR_REGISTRY)
```
Comment (corinnekrych) [inline make/Makefile.dev]: @baijum Is your PR dependent on another PR? where is DEVOPSCONSOLE_OPERATOR_REGISTRY defined?

Comment (baijum) [reply make/Makefile.dev]: I had another PR which introduced it. I merged today morning.

Comment (baijum) [reply make/Makefile.dev]: Fixed 50c2e205deee1acbb6e505943c9f4743443d3a29

File: make/Makefile.dev (hunk 1 above)
Comment (baijum) [reply make/Makefile.dev]: Done. 28d030818095cea6383d25c31dfccef38b39cfde

**Pull request merged**